### DISCUSSION TITLE
default sass does not compile

### DISCUSSION THREAD

**Discussion by joshuacox** [gatsbyjs/gatsby#28675] [Help] [answered]
Following the [official gatsby docs on sass](https://www.gatsbyjs.com/docs/how-to/styling/sass/) results in this error:

```
 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed


$primary-color: #333
             ^
      Expected ";".
  ╷
2 │ $primary-color: #333
  │               ^
  ╵
  stdin 2:15  root stylesheet
      in /tmp/sass-test/src/styles/sassy.sass (line 2, column 15)

File: src/styles/sassy.sass

failed Building development bundle - 2.325s
```

full asciinema recording of steps to reproduce [sass-test.asciinema.gz](https://github.com/gatsbyjs/gatsby/files/5711351/sass-test.asciinema.gz)

There is also a [discussion](https://github.com/gatsbyjs/gatsby/issues/27754) about using sass instead of node-sass that results in the same, with another asciinema file to reproduce the results
[sass-test2.asciinema.gz](https://github.com/gatsbyjs/gatsby/files/5711400/sass-test2.asciinema.gz).

The sass file is no different than the basic sass file from the [sass docs](https://sass-lang.com/guide)

**Comment by vladar** [comment, marked as answer]
I think it's the same as https://github.com/gatsbyjs/gatsby/issues/27842#issuecomment-722998875

You need to enable indented syntax to make it work:

```js
module.exports = {
  /* Your site config here */
  plugins: [
    { resolve: `gatsby-plugin-sass`, options: { indentedSyntax: true } },
  ],
}
```

### New Major

Also, the new major version of the `gatsby-plugin-sass` will be published on `Jan 5, 2021` that will be based on `sass`. See #27991

You can try it today by installing `gatsby-plugin-sass@next` alongside with `sass`

**Comment by joshuacox** [comment]
shouldn't this be documented as the sass file given in the official documentation does not work without this option enabled?

**Comment by joshuacox** [comment]
I've tested this and sure enough the option does work with the node-sass route.

However with the `gatsby-plugin-sass@next` version I get:

```
undefined failed

Invalid options object. Sass Loader has been initialized using an options object that does not match the API schema.
 - options has an unknown property 'indentedSyntax'. These properties are valid:
   object { implementation?, sassOptions?, additionalData?, sourceMap?, webpackImporter? }
```

in which case switching back to the normal invocation in `gatsby-config.js` did the trick.

```
`gatsby-plugin-sass`,
```

**Reply by jgthms** [reply]
You now have to write:

```js
{
  resolve: "gatsby-plugin-sass",
  options: {
    sassOptions: {
      indentedSyntax: true,
    },
  },
},
```
